src/GeneralStateTestsFiller/stCreate2/CREATE2_HighNonceDelegatecallFiller.yml

```diff
@@ -0,0 +1,588 @@
+    # - Contract A (Nonce=0xfffffffffffffffe) code calls CREATE2 Contract B (Nonce=0xfffffffffffffffe), contract create succeeds
+      - ':label a_max_nonce_minus_1_call_code_create2 :abi f(uint,uint,uint,uint) 0x01 0xfffffffffffffffe 0xfffffffffffffffe 0x1'
+    # - Contract A (Nonce=0xfffffffffffffffe) code calls CREATE2 Contract B (Nonce=0xffffffffffffffff), contract create succeeds
+      - ':label a_max_nonce_minus_1_call_code_create2 :abi f(uint,uint,uint,uint) 0x01 0xfffffffffffffffe 0xffffffffffffffff 0x1'
+    # - Contract A (Nonce=0xfffffffffffffffe) calls CREATE2 Contract B (Nonce=0xfffffffffffffffe), contract create succeeds
+      - ':label a_max_nonce_minus_1_call_create2 :abi f(uint,uint,uint,uint) 0x02 0xfffffffffffffffe 0xfffffffffffffffe 0x1'
+    # - Contract A (Nonce=0xfffffffffffffffe) calls CREATE2 Contract B (Nonce=0xffffffffffffffff), contract create fails
+      - ':label a_max_nonce_minus_1_call_max_create2 :abi f(uint,uint,uint,uint) 0x02 0xfffffffffffffffe 0xffffffffffffffff 0x1'
+
+    # CREATE2 + Caller (A) Max Nonce Test Cases
+    # - Contract A (Nonce=0xffffffffffffffff) delegate calls CREATE2 Contract B (Nonce=0xfffffffffffffffe), contract create fails
+      - ':label a_max_nonce_delegate_call_create2 :abi f(uint,uint,uint,uint) 0x00 0xffffffffffffffff 0xfffffffffffffffe 0x1'
+    # - Contract A (Nonce=0xffffffffffffffff) delegate calls CREATE2 Contract B (Nonce=0xffffffffffffffff), contract create fails
+      - ':label a_max_nonce_delegate_call_create2 :abi f(uint,uint,uint,uint) 0x00 0xffffffffffffffff 0xffffffffffffffff 0x1'
+    # - Contract A (Nonce=0xffffffffffffffff) code calls CREATE2 Contract B (Nonce=0xfffffffffffffffe), contract create fails
+      - ':label a_max_nonce_call_code_create2 :abi f(uint,uint,uint,uint) 0x01 0xffffffffffffffff 0xfffffffffffffffe 0x1'
+    # - Contract A (Nonce=0xffffffffffffffff) code calls CREATE2 Contract B (Nonce=0xffffffffffffffff), contract create fails
+      - ':label a_max_nonce_call_code_create2 :abi f(uint,uint,uint,uint) 0x01 0xffffffffffffffff 0xffffffffffffffff 0x1'
+    # - Contract A (Nonce=0xffffffffffffffff) calls CREATE2 Contract B (Nonce=0xfffffffffffffffe), contract create succeeds
+      - ':label a_max_nonce_call_create2 :abi f(uint,uint,uint,uint) 0x02 0xffffffffffffffff 0xfffffffffffffffe 0x1'
+    # - Contract A (Nonce=0xffffffffffffffff) calls CREATE2 Contract B (Nonce=0xffffffffffffffff), contract create fails
+      - ':label a_max_nonce_call_max_create2 :abi f(uint,uint,uint,uint) 0x02 0xffffffffffffffff 0xffffffffffffffff 0x1'
+
+    gasLimit:
+      - 70000000
+    gasPrice: 10
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: 'b94f5374fce5edbc8e2a8697c15331677e6ebf0b'
+    value:
+    - 0
+
+  expect:
+    - indexes:
+       data:
+       - ':label a_max_nonce_delegate_call_create'
```

Comment:
Perhaps better `A_MaxNonce_DelegateCall_B_MaxNonce_Create`

Reply:
Change the labels for all test cases to this format.